alerter.py

```diff
@@ -5,7 +5,7 @@ def network_alert_stub(celcius):
     # Return 200 for ok
     # Return 500 for not-ok
     # stub always succeeds and returns 200
-    return 200
+    return 500

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.